README.md

```diff
@@ -9,13 +9,28 @@ freezes, being able to actually _use_ your computer when the build is happening.
 
 ## Table of contents
 
+- [State of Project Development](#state-of-project-development)
 - [Supported Actions](#supported-actions)
 - [Setup](#setup)
 - [Demo](#demo)
 - [3rd-party IntelliJ Mainframer Plugin](#3rd-party-intellij-mainframer-plugin)
 - [How to Run Tests (for contributors)](#how-to-run-tests-for-contributors)
 - [License](#license)
 
+## State of Project Development
+
+Version `3.x` is in active development, current stable version is `2.x` and recommended for production use.
+
+Since we store everything on GitHub, we recommend viewing documentation for specific version using git tags (see [releases page](https://github.com/gojuno/mainframer/releases)).
```

Comment:
This line is mostly useless since people are actually interested in `2.x`. I suggest replacing it with this and merging with the first line:

```
You can view `2.x` version using [the Git tag](TAG).
```